03_launch_mgmt_cluster.sh

```diff
@@ -464,6 +464,45 @@ function start_management_cluster () {
   fi
 }
 
+build_ipxe_firmware() {
+# Build iPXE firmware during deployment (only available on ubuntu)
+    # vars with CENV_ARG postfix are container environment variable arguments
+    # and only used to pass the env var to the containers
+    export IPXE_ENABLE_TLS_CENV_ARG="IPXE_ENABLE_TLS='false'"
+    export IPXE_ENABLE_IPV6_CENV_ARG="IPXE_ENABLE_IPV6='false'"
+    CERTS_MOUNTS=""
+    if [[ "${BUILD_IPXE}" = "true" ]]; then
```

Comment:
I would suggest reversing this so we check if BUILD_IPXE!=true and return early in that case.

Reply:
Done